arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -3751,6 +3951,38 @@
       },
       "description": "The content hash."
     },
+    "RetryHistory": {
+      "type": "object",
+      "properties": {
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the start time."
```

Comment:
Do we need to say "Gets the ...." here? Agreed this is a get, but we should simply describe the property here.
Similar comment for other properties below.